.github/scripts/trymerge.py

```diff
@@ -931,7 +931,7 @@ def merge(pr_num: int, repo: GitRepo,
     initial_commit_sha = pr.last_commit()['oid']
     if force:
         return pr.merge_into(repo, dry_run=dry_run, force=force, comment_id=comment_id)
-    if (datetime.utcnow() - pr.last_pushed_at()).days > stale_pr_days:
+    if (datetime.utcnow() - pr.last_pushed_at()).days > stale_pr_days and not can_skip_internal_checks(pr, comment_id):
```

Comment:
Should it simply be moved to force step?

Reply:
`force` would ignore failing checks on the PR. Is that acceptable?

Reply:
Yes